plugin-packages/multimedia/test/src/video-component.spec.ts

```
@@ -0,0 +1,394 @@
+import { generateGUID, Player, spec } from '@galacean/effects';
+import { VideoComponent } from '@galacean/effects-plugin-multimedia';
+
+export interface VideoCompositionOptions {
+  duration: number,
+  endBehavior: spec.EndBehavior,
+  id: string,
+  videos: spec.AssetBaseOptions[],
+  start: number,
+  options: spec.VideoContentOptions,
+}
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)